README.md

```diff
@@ -268,9 +274,9 @@ The `sendToAnalytics()` function uses the [`navigator.sendBeacon()`](https://dev
 import {onCLS, onINP, onLCP} from 'web-vitals';
 
 function sendToAnalytics(metric) {
-  // Replace with whatever serialization method you prefer.
-  // Note: JSON.stringify will likely include more data than you need.
-  const body = JSON.stringify(metric);
+  // Replace with whatever serialization method you prefer including
```

Comment:
I think changing the code to list only the specific data being sent is good. But maybe include some more bits of data in the example so it's more clear how someone else might customize this for their needs?

```js
const body = JSON.stringify({
  name: metric.name,
  value: metric.value,
  id: metric.id,
  
  // Include additional data as needed...
});
```


I'd also rephrase the comment to something like: "Replace with whatever specific data you want to capture."

"Replace with whatever metric data you need

Reply:
I think we can drop the comment and just stick with the "Include additional data as needed..." comment.

Did that.